eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -20,9 +20,33 @@ function Get-Suppression {
 
   # -NoEnumerate to prevent single-element arrays from being collapsed to a single object
   # -AsHashtable is closer to raw JSON than PSCustomObject
-  $suppressions = npx get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
+  $suppressions = npm exec --no -- get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
 
-  return $suppressions ? $suppressions[0] : $null;
+  if ($LASTEXITCODE -ne 0) {
+      LogError "Failure running 'npm exec get-suppressions'"
+      LogJobFailure
+      exit 1
+  }
+
+  $suppression = $suppressions ? $suppressions[0] : $null
```

Comment:
It is not immediately obvious to me why we take `[0]` and ignore other entries / assume they are absent. Maybe add a line of comment explaining this?

Reply:
Good point.  For now we just take the first matching suppression, but I want to revist later what we should do if multiple match.

Reply:
Added comment and created issue